common/test/acceptance/pages/lms/teams.py

```diff
@@ -138,6 +138,12 @@ def topic_cards(self):
         return self.q(css=TOPIC_CARD_CSS).results
 
     @property
+    def topic_team_counts(self):
```

Comment:
Note that there is a verify_topic_team_count method in TeamsPage. 

I had to put some of these methods that are related to subview in TeamsPage because I wanted to test the refresh behavior after something has been marked stale. I found that using the more scoped objects caused them to be visited through their URLs, which meant that I couldn't reproduce the original bugs. I needed to navigate through the breadcrumbs and then verify things on the page without going through the routing URLs.

If you've found another way to do this that allows the methods to be in a more logical place, I'd be happy to have them moved. But I want to make sure that the tests fail without the "stale" logic in place.